exp/api/v1alpha3/azuremanagedcontrolplane_types.go

```diff
@@ -45,7 +47,7 @@ type AzureManagedControlPlaneSpec struct {
 	// AdditionalTags is an optional set of tags to add to Azure resources managed by the Azure provider, in addition to the
 	// ones added by default.
 	// +optional
-	AdditionalTags map[string]string `json:"additionalTags,omitempty"`
+	AdditionalTags infrav1.Tags `json:"additionalTags,omitempty"`
```

Comment:
This is a breaking change to the API :(.

Reply:
Let's mark these as "action required" in the release note. I think it's acceptable to have breaking changes at this point given that it's 1) an alpha api 2) an experimental feature

Reply:
This might actually not be a breaking change from a yaml spec perspective? since infrav1.Tags is defined as `type Tags map[string]string`

Reply:
Still should mark it as action required just in case; would technically be a break for anyone importing the types (likely N=0, in fairness).